core/modules/server.py

```diff
@@ -793,8 +806,7 @@ def supportfs(self):
 
     print('* LVM partitions:')
     for partition in diskinfo['lvm']['partitions']:
-        print('  - Partition name: %s (%d, %d)' % \)
-            (partition['name'], partition['major'], partition['minor'])

```

Comment:
It cause an error when I used the --dev arg

Reply:
Thank you for your contribution